chainlit_app/app.py

```diff
@@ -0,0 +1,66 @@
+import services.chat_service
+import services.feedback_service
+import services.history_service
+from chainlit import Message, on_chat_start, on_message
+
+
+@on_chat_start
+async def chat_start() -> None:
+    """
+    Triggered when the chat starts. Welcomes the user
+    and gives options to fetch history.
+    """
+    await Message(
+        content="Welcome! You can ask questions on health services delivery."
+    ).send()
+
+
+@on_message
+async def main(message: Message) -> None:
+    """
+    Handles user messages and provides a response from the chat API.
+    """
+    user_message = message.content
+
+    response = services.chat_service.get_chat_response(user_message)
+    if "error" in response:
```

Comment:
We should probably use a pydantic class for the return object and have a `status` field for `error`? Or use enum

Reply:
As you the API is not integrated, I don't even know the response model, can we add this after the API can integrated